wcfsetup/install/files/lib/data/user/avatar/UserAvatar.class.php

```diff
@@ -14,13 +15,14 @@
  * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
  * @package WoltLabSuite\Core\Data\User\Avatar
  *
- * @property-read   int $avatarID       unique id of the user avatar
- * @property-read   string $avatarName     name of the original avatar file
- * @property-read   string $avatarExtension    extension of the avatar file
- * @property-read   int $width          width of the user avatar image
- * @property-read   int $height         height of the user avatar image
- * @property-read   int|null $userID         id of the user to which the user avatar belongs or null
- * @property-read   string $fileHash       SHA1 hash of the original avatar file
+ * @property-read int $avatarID unique id of the user avatar
+ * @property-read string $avatarName name of the original avatar file
+ * @property-read string $avatarExtension extension of the avatar file
+ * @property-read int $width width of the user avatar image
+ * @property-read int $height height of the user avatar image
+ * @property-read int|null $userID id of the user to which the user avatar belongs or null
+ * @property-read string $fileHash SHA1 hash of the original avatar file
+ * @property-read int $hasWebP `1` if there is a WebP variant, else `0`
```

Comment:
This might be related to the PSR-12 changes, but this should not have happened. Can you check what caused this reformatting? Maybe it should / needs to be applied to all files before.

Reply:
That was my work, I manually removed the extra spaces.